server/lib/Room.js

```diff
@@ -1378,8 +1378,16 @@ class Room extends EventEmitter
 				)
 					throw new Error('peer not authorized');
 
-				// Spread to others
-				this._notification(peer.socket, 'moderator:mute', null, true);
+				const { peerId } = request.data;
+
+				const mutePeer = this._peers[peerId];
+
+				if (!mutePeer)
+					throw new Error(`peer with id "${peerId}" not found`);
+
+				this._notification(mutePeer.socket, 'moderator:mute');
+
+				mutePeer.close();
```

Comment:
It seems a bit rude to throw the peer out of the meeting ;)

Reply:
Seems you beat me to it!

Reply:
Well if he's very noisy, he deserves it! :P

Already fixed, i'll test right now and let you know.